submissions/m-ruslan/js-dom/app.js

```diff
@@ -0,0 +1,86 @@
+document.addEventListener("DOMContentLoaded", () => {
+  console.log("DOM is loaded");
+  loadPageData("info");
+
+  for (i in pageSections) {
+    document
+      .querySelector("#" + pageSections[i].id)
+      .addEventListener("click", (event) => {
+        console.log(event.path[0].id);
```

Comment:
Here is what I see when I open the site in Firefox and click on menu items: 

![image](https://user-images.githubusercontent.com/17703790/100517328-8ddd2a00-3192-11eb-8c09-cc9a2f488768.png)

The problem is that `MouseEvent.path` is not a standard property and is not supported in many browsers. Don you need this particular property or maybe, for example, `Event.currentTarget` would be ok?

Reply:
Thank you
I've changed to `Event.target` . I've thried `Event.currentTarget` at first, but when i added the event deligation my code hasn't worked with `Event.currentTarget` but it works with `Event.target` (currentTarget was `.nav-container` and target was `a`)